src/python/WMCore/Services/RucioConMon/RucioConMon.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# coding=utf-8
+"""
+Rucio Consistency Monitor Service class developed on top of the native WMCore
+Service APIs providing custom output and handling as necessary for the
+CMS Workload Management system
+"""
+
+from __future__ import division, print_function, absolute_import
+from future import standard_library
+
+from urllib.parse import urlencode
+
+import json
+import logging
+
+from WMCore.Services.Service import Service
+standard_library.install_aliases()
```

Comment:
I believe the order of these imports will matter. So this `install_aliases()` line should go right in the next line after you import it (and before you import urllib.parse).

Reply:
> So this install_aliases() line should go right in the next line after you import it (and before you import urllib.parse)

I'll fix it as you say, and this is how it was until earlier, but just to warn you - this will break another pylint requirement : 
> E402 module level import not at top of file

Reply:
The only way to work around that one would be disabling it, but I usually don't disable any pylint rule. So it's fine like this.